src/main/java/org/umssdiplo/automationv01/core/managepage/navigationbar/NavigationBar.java

```diff
@@ -3,43 +3,58 @@
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.umssdiplo.automationv01.core.managepage.BasePage;
-import org.umssdiplo.automationv01.core.managepage.accidenteincidentmenu.AccidentEIncidentMenu;
+import org.umssdiplo.automationv01.core.managepage.organizationalstructuremenu.EmployeeList;
+import org.umssdiplo.automationv01.core.managepage.organizationalstructuremenu.ListDepartmentPage;
 import org.umssdiplo.automationv01.core.managepage.organizationalstructuremenu.OrganizationalStructureMenu;
 import org.umssdiplo.automationv01.core.managepage.storage.Storage;
 import org.umssdiplo.automationv01.core.managepage.storagemenu.StorageMenu;
 import org.umssdiplo.automationv01.core.managepage.reportssubmenu.ReportsSubMenu;
 import org.umssdiplo.automationv01.core.utils.CommonEvents;
 
+import javax.smartcardio.CommandAPDU;
+
 public class NavigationBar extends BasePage {
     @FindBy(id = "menu-reports")
     private WebElement reportsMenu;
 
     @FindBy(id = "menu-organizational")
-    private WebElement organizationalMenu;
+    private WebElement activeMenu;
+
+    @FindBy(xpath = "//span[contains(text(), 'Departamentos')]")
+    private WebElement activeSubmenu;
 
-    @FindBy(id = "accident-incident-button")
-    private WebElement accidentIncidentMenu;
+    @FindBy(id = "sub-menu-employees")
+    private WebElement employeeSubMenu;
 
     @FindBy(id = "menu-storage")
     private WebElement storageMenu;
 
     public NavigationBar() {
+        CommonEvents.customWait(600);
         CommonEvents.isVisible(reportsMenu);
     }
 
     public OrganizationalStructureMenu clickOrganizationalStructure() {
-        CommonEvents.clickButton(organizationalMenu);
```

Comment:
I think it is not necessary to delete this line, because we it is executing the action